practice-app/website/models.py

```diff
@@ -46,3 +46,37 @@ def serialize(self):
            'creator_user'           : self.creator_user,
            'sport'                  : self.sport
        }
+
+
+
+
+# Corresponds to a discussion page for an event
+class DiscussionPost(db.Model):
+    id = db.Column(db.Integer, primary_key=True)

```

Comment:
does primary key result in every event has only one discussion post?

Reply:
The internal structure may look a bit complicated, yet it works as intended.